Documentation/gettingstarted/local-redirect-policy.rst

```diff
@@ -412,21 +412,19 @@ security credentials for pods.
 
       $ helm repo add uswitch https://uswitch.github.io/kiam-helm-charts/charts/
       $ helm repo update
-      $ helm template kiam uswitch/kiam > kiam.yaml
 
-  - If you see an error like "request blocked by whitelist-route-regexp" while
-    running requests to the metadata server, then you may need to whitelist the
-    metadata requests by passing the below argument to the ``kiam-agent Deamonset``.
+  - You may see an error like "request blocked by whitelist-route-regexp" while
+    running requests to the metadata server once the kiam-agent pods are deployed.
+    In such cases, you may need to whitelist the metadata requests.
 
     .. code-block:: bash
 
-        $ sed -i '/args:/a \ \ \ \ \ \ \ \ \ \ \ \ - --whitelist-route-regexp=meta-data' kiam.yaml
+        $ helm template --set agent.extraArgs.whitelist-route-regexp=meta-data kiam uswitch/kiam > kiam.yaml
```

Comment:
Reading the overall list of items here, we have 

- Deploy
  - Instructions add the repository, but don't actually deploy
- You may see an error....
   - Template instructions
- ...
- Apply the configuration
  - Finally, the step where we deploy.

Which seems a bit odd in terms of ordering if you just read through?

Also, now that we don't need to run the extra `sed` commands, we could just do `helm install` directly with this option?

I'm wondering whether it's even worth mentioning things like the "request blocked while whitelist-out-regexp" bullet point above or the "iptables" bullet point below if we just tell people to install with the following command:

```
   helm install --set agent.host.iptables=false --set agent.extraArgs.whitelist-route-regexp=meta-data kiam uswitch/kiam
```

We can always add items afterwards to explain why each setting is configured like this, that's fine but I feel like we should provide the exact set of options that we expect users to deploy with to have the best chance of success. Then if a user deviates from these instructions, they can consult the documentation and see exactly which options we recommended.